sequencer/src/genesis.rs

```diff
@@ -1254,7 +1255,7 @@ mod test {
     }
 
     #[test]
-    fn test_marketplace_and_fee_upgrade_toml() {
+    fn test_fee_and_epoch_upgrade_toml() {
         let toml = toml! {
             base_version = "0.1"
             upgrade_version = "0.2"
```

Comment:
would be nice if you can include the epoch_height and epoch_start_block here as well